### ISSUE TITLE
Technique H46 has almost no AT support

### ISSUE THREAD

**Issue by awkawk** [w3c/wcag#128]
H46 doesn't really work in modern AT:

http://www.powermapper.com/tests/screen-readers/media/embed-noembed-after/
http://www.powermapper.com/tests/screen-readers/media/embed-noembed-inside/

Some WCAG failures trigger fewer problems than this sufficient technique.

Proposed Change:
Remove technique.

Worth noting that an alternative technique based on FIGCAPTION does seem to work well:

http://www.powermapper.com/tests/screen-readers/media/embed-figcaption/

**Comment by joshueoconnor** [comment]
+1 to removing or updating,

**Comment by joshueoconnor** [comment]
CfC: Drop advisory technique H46 went to group on 1st March. End of call Friday 4th 2.30 EST.

**Comment by joshueoconnor** [comment]
@mraccess77 Has commented that this does actually work on iOS and as that represents a valid use case, we will retain this technique.

**Comment by awkawk** [comment]
Preparing to mark as deferred due to inactivity and group resource constraints (https://www.w3.org/WAI/GL/wiki/Handling_Issues).
